### ISSUE TITLE
[WebGL2] The outputs don't match the expected values for Concatenation operation

### ISSUE THREAD

**Issue by Christywl** [intel/webml-polyfill#132]
**Test Env:**
Chromium Version : nightly build 70.0.3503.0 (revision 3dd7088)
OS : Mac OS
webml-polyfill: commit 777b85f

**Expected Result:**
Tests should pass.

**Actual Result:**
The actual outputs don't match the expected values.
- Concatenation Test for 4-D input with axis being 0:
   expected:
```
[101.0, 102.0, 103.0, 104.0, 105.0, 106.0, 107.0, 108.0,     
 109.0, 110.0, 111.0, 112.0, 113.0, 114.0, 115.0, 116.0,      
 201.0, 202.0, 203.0, 204.0, 205.0, 206.0, 207.0, 208.0,
 209.0, 210.0, 211.0, 212.0, 213.0, 214.0, 215.0, 216.0]
```
actual:

```
[0, 0, 0, 0, 0, 0, 0, 0,
 0, 0, 0, 0, 0, 0, 0, 0,
 0, 0, 0, 0, 0, 0, 0, 0,
 0, 0, 0, 0, 0, 0, 0, 0]
```

- Concatenation Test for 4-D input with axis being 1:
expected:
```
[101.0, 102.0, 103.0, 104.0, 105.0, 106.0, 107.0, 108.0,
 201.0, 202.0, 203.0, 204.0, 205.0, 206.0, 207.0, 208.0,
 109.0, 110.0, 111.0, 112.0, 113.0, 114.0, 115.0, 116.0,
 209.0, 210.0, 211.0, 212.0, 213.0, 214.0, 215.0, 216.0]
```
actual:
```
[101.0, 102.0, 201.0, 202.0,
 103.0, 104.0, 203.0, 204.0,
 105.0, 106.0, 205.0, 206.0,
 107.0, 108.0, 207.0, 208.0,
 109.0, 110.0, 209.0, 210.0,
 111.0, 112.0, 211.0, 212.0,
 113.0, 114.0, 213.0, 214.0,
 115.0, 116.0, 215.0, 216.0]
```
- Concatenation Test for 4-D input with axis being 2
expected:
```
[101.0, 102.0, 103.0, 104.0,
 201.0, 202.0, 203.0, 204.0,
 105.0, 106.0, 107.0, 108.0,
 205.0, 206.0, 207.0, 208.0,
 109.0, 110.0, 111.0, 112.0,
 209.0, 210.0, 211.0, 212.0,
 113.0, 114.0, 115.0, 116.0,
 213.0, 214.0, 215.0, 216.0]
```
actual:
```
[101.0, 102.0, 201.0, 202.0,
 103.0, 104.0, 203.0, 204.0,
 105.0, 106.0, 205.0, 206.0,
 107.0, 108.0, 207.0, 208.0,
 109.0, 110.0, 209.0, 210.0,
 111.0, 112.0, 211.0, 212.0,
 113.0, 114.0, 213.0, 214.0,
 115.0, 116.0, 215.0, 216.0]
```
**How to reproduce:**
1. Launch chromium and disable the WebML
2. Run tests: access to https://brucedai.github.io/nt/test/concatenation_012.html?backend=webgl2

**Comment by Wenzhao-Xiang** [comment]
Looks #56 ,

>**2.  CONCATENATION in WebGL2 version only supports concatenating channel axis(Don't support batch, height and width concatenation).**

WebGL2 4D shape is based on 3D shape. Therefore, CONCATENATION in WebGL2 version only supports concatenating channel axis,too.

It will be enhanced if needed.

**Comment by GreyZzzzzzXh** [comment]
This bug has been fixed by replacing webgl2 with tfjs-core (PR #297).

**Comment by Christywl** [comment]
All the tests passed on webgl backend on nightly build 6540a81.
